### DISCUSSION TITLE
Support Swift Package Manager

### DISCUSSION THREAD

**Discussion by sharezy** [Tencent/MMKV#686] [Q&A] [answered]
Do we have a plan to support SPM?

**Comment by lingol** [comment, marked as answer]
Tried and aborted. You can search issues for details.

发自我的iPhone

在 2021年5月20日，下午10:56，sharezy ***@***.***> 写道：

﻿

Do we have a plan to support SPM?

**Reply by sharezy** [reply]
I found this. I don't know if it helps.

https://forums.swift.org/t/override-for-unsafeflags-in-swift-package-manager/45273/5

**Reply by lingol** [reply]
NO, it didn't. An open-source project can't possibly have any **local** repo.

**Reply by sharezy** [reply]
Yes,but it's a way to integrate MMKV to my project by SPM, thanks for your reply🙂
